src/core/HorizontalTabs/HorizontalTab.tsx

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/tabs.css';
+import { CommonProps } from '../utils/props';
+
+export type HorizontalTabProps = {
+  /**
+   * The main label shown in the tab.
+   */
+  label?: React.ReactNode;
+  /**
+   * Secondary label shown below the main label.
+   * Only shown if `size` of `HorizontalTabs` is set to large.
+   */
+  sublabel?: React.ReactNode;
+  /**
+   * Svg icon shown before the labels.
+   */
+  icon?: JSX.Element;
```

Comment:
Maybe lets do `startIcon`, so its obviuos where its put (and maybe we will need endIco in future :DD)

Reply:
Done